chapters/glossary/glossary.rst

```diff
@@ -0,0 +1,78 @@
+@@ -1,75 +0,0 @@
+Glossary
+============
+
+The following terminology is commonly used within the Jina framework. Jina is an active development, meaning these terms may be updated and refined regularly as new and improved features are added.
+
+.. glossary::
+
+    Chunk
+        Chunks are individual semantic units of a Document. This occurs within the Segmenter class of Executors. Examples of individual units would be sentences from large documents or pixel patches from an image. `For further information see the Understand Jina Recursive Document Representation guide. <https://docs.jina.ai/chapters/traversal/index.html?highlight=recursive>`_
+    Client
+        A Python client that connects to a remote Jina gateway.
+    Container
+        A Docker container image is a lightweight, standalone, executable package of software that includes everything needed to run an application: code, runtime, system tools, system libraries and settings.
+    Crafter
+        Crafters are a class of Executor whose purpose is to alter a Document to match a certain requirement. This could include tasks such as resizing images or reducing all text in a document to lowercase.
+    Document
+        Documents are the most basic data types available within Jina. Anything you want to search for within Jina is considered a document, this could include images, sounds clips or text documents.
+    Driver
+        Drivers wrap an Executor and interprets network traffic into a format the Executor can understand. All Executors must have at least one driver. A Driver is further wrapped by a Pea.
+    Embedding space
+        Embedding space is the vector space in which the data is embedded after dimensionality reduction. Depending on how the space is created, semantically similar items are put together and dissimilar items are kept far apart. Position (distance and direction) in the vector space can encode semantics in a good embedding. `reference_1 <https://link.springer.com/referenceworkentry/10.1007%2F978-0-387-73003-5_573#:~:text=Embedding%20space%20is%20the%20space,than%20of%20the%20ambient%20space.>`_
+    Encoder
+        Encoders are a class of Executor whose purpose is to take a variable-length input and output a fixed-length semantic embedding vector. This is achieved by passing the input to a pre-created language model which returns a fixed length context specific vector.
+    Evaluator
+        Evaluators are a class of Executor whose purpose is to provide advanced evaluation metrics on the performance of a search system. Distance metrics such as cosine distance and euclidean distance. As well evaluation metrics such as precision, and recall between the results and the ground truth (provided by user) can be computed.
```

Comment:
This can be broken into two parts:
Evaluators can compute:

- Distance metrics, for instance cosine or euclidean distance

- Precision and recall of results as compared against the user-provided ground truth

Reply:
Is it good practice to have bullet points on a glossary?

Reply:
Despite what I said in the tech writing workshop, I think glossary term should be short enough not to have bullet points.